app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionBlockingFragment.kt

```diff
@@ -22,6 +23,7 @@ class TrackingProtectionBlockingFragment :
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        category_redirect_trackers.isVisible = FeatureFlags.etpCookiePurging
```

Comment:
I think the same applies as in https://github.com/mozilla-mobile/fenix/pull/15139/files#r490443152

Reply:
That link isn't working for me 🤔

Reply:
Np, it was related to the Emely's comment, but `isVisible` is applying `gone` underneath, so ignore the the comment, this is good :)